src/test/java/org/dependencytrack/notification/publisher/JiraPublisherTest.java

```diff
@@ -34,20 +34,18 @@
 import static org.dependencytrack.model.ConfigPropertyConstants.JIRA_URL;
 import static org.dependencytrack.model.ConfigPropertyConstants.JIRA_USERNAME;
 
-public class JiraPublisherTest extends AbstractWebhookPublisherTest<JiraPublisher> {
+class JiraPublisherTest extends AbstractWebhookPublisherTest<JiraPublisher> {
```

Comment:
Executing this test does no actually run the methods annotated with `@Override`, but they run their super class' methods instead. You can observe this by setting a breakpoint on the `verify` calls - the breakpoint won't trigger.

IIRC this is a behavior change in JUnit 5, in that it no longer runs `@Test` methods from superclasses. You'll need to remove `@Test` from methods in `AbstractPublisherTest`, and add them to the `@Override` methods instead.

Reply:
It seems there's now an error within the new `baseTestInformWithTemplateInclude` (I renamed the base class methods), running from within `SendMailPublisherTest`. I can't tell whether this is a legitimate error.